catalog/ui/search-ui/standard/pom.xml

```diff
@@ -214,9 +214,16 @@
                 <groupId>org.asciidoctor</groupId>
                 <artifactId>asciidoctor-maven-plugin</artifactId>
                 <version>1.5.2</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.asciidoctor</groupId>
+                        <artifactId>asciidoctorj-pdf</artifactId>
+                        <version>1.5.0-alpha.6</version>
```

Comment:
I haven't been on the asciidoctor-pdf docs in a while, but as I understand, it's still technically "experimental."